guides/17_Migration_to_3_0.md

```diff
@@ -163,6 +163,117 @@ handle them please remove.
 - StateMachineTransitionFailed
 - AudioDeviceSwitched
 
+## AudioVideo events
+
+We have removed below `AudioVideo` events in v3.
+
+- videoSendHealthDidChange
+- videoSendBandwidthDidChange
+- videoReceiveBandwidthDidChange
+- videoNotReceivingEnoughData
+- estimatedDownlinkBandwidthLessThanRequired
+
+In 3.x you can still replecate these events funtionality through `metricDidReceive`. However `estimatedDownlinkBandwidthLessThanRequired` can not be easily replicated.
```

Comment:
Could we clarify that there is no action needed if they do not use any of these events?

Reply:
I updated to mention you can use priority-based downlink to manage videos instead of `videoNotReceivingEnoughData` and `estimatedDownlinkBandwidthLessThanRequired `